resilience_stats/api.py

```diff
@@ -131,6 +131,26 @@ def run_outage_sim_task(scenariomodel_id, run_uuid, bau):
 
     try:
         ResilienceModel.objects.filter(scenariomodel_id=scenariomodel_id).update(**results)
+        results = {'avoided_outage_costs_us_dollars': results['avoided_outage_costs_us_dollars']}
+        FinancialModel.objects.filter(run_uuid=run_uuid).update(**results)
+        deprecation_message = ("'Saving avoided outage costs within the Financial model, accessed from the /results endpoint,"
```

Comment:
@t-kwasnik I don't think that this message will make sense from a user's perspective since they know nothing of the database back-end. What do you think of putting a different deprecation message about the `avoided_outage_costs_us_dollars` moving from the results to the resilience_stats in the validator? (To replace this deprecation message.)